RELEASE_PROCESS.md

```diff
@@ -2,18 +2,32 @@
 Copyright 2021 Ocean Protocol Foundation
 SPDX-License-Identifier: Apache-2.0
 -->
+
 # The aquarius Release Process
 
+## Step 1: Bump version and push changes
+
 - Create a new local feature branch, e.g. `git checkout -b feature/bumpversion-to-v0.2.5`
 - Use the `bumpversion.sh` script to bump the project version. You can execute the script using {major|minor|patch} as first argument to bump the version accordingly:
   - To bump the patch version: `./bumpversion.sh patch`
   - To bump the minor version: `./bumpversion.sh minor`
   - To bump the major version: `./bumpversion.sh major`
 - Commit the changes to the feature branch.
+
+  `git commit -m "release 2.2.9"`
+
 - Push the feature branch to GitHub.
+
+  `git push origin feature/bumpversion-to-v0.2.5`
+
+## Step 2: Merge changes to develop branch
+
 - Make a pull request from the just-pushed branch.
 - Wait for all the tests to pass!
 - Merge the pull request into the `develop` branch.
```

Comment:
maybe "main" branch ?

Reply:
I agree. It should be `main`. CC: @akshay-ap